.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@agentofreality @danielgerlag
\ No newline at end of file
+* @drasi-project/drasi-core-engineers
```

Comment:
@ruokun-niu I created a new team @drasi-project/maintainers-core, so lets use that team